src/index.html

```diff
@@ -9,10 +9,62 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
Stylesheets with a .scss extension cannot be linked directly in HTML. You should compile them to .css files first, then link the compiled .css file here.